janusgraph-hadoop-parent/pom.xml

```diff
@@ -191,13 +191,6 @@
             <artifactId>logback-classic</artifactId>
         </dependency>
         <!-- End logging backends. -->
-
-        <!-- TESTING -->
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <scope>test</scope>
-        </dependency>
```

Comment:
Is this not needed because janusgraph-test is in the dependency list?

Reply:
Correct